pom.xml

```diff
@@ -21,6 +21,12 @@
         <configuration>
           <parallel>suites</parallel>
           <redirectTestOutputToFile>true</redirectTestOutputToFile>
+          <includes>
+            <include>**/Test*.java</include>
+            <include>**/*IT.java</include>
+            <include>**/*Test.java</include>
```

Comment:
Is this and the following pattern important, since `**/Test*.java` already seems to cover them?

Reply:
`**/Test*.java` handles files where `Test` is the prefix, `**/*Test.java` handles files where `Test` is the suffix, so yes it is important. We have both.